src/api/tokenRefresher.js

```diff
@@ -0,0 +1,30 @@
+import { apiHandler } from './apiHandler';
+
+export const refreshToken = (refreshToken) =>
+    apiHandler('/refresh-token', 'POST', { refreshToken });
+
+export const callApiWithRefresher = async (api, params) => {
```

Comment:
bad naming, withRefresher is also shouldn't be mentioned

Reply:
Should we call it `callApiWithAutomaticTokenRefresh` or do you have a different suggestion?

Reply:
token refresh is a side interceptor, i don't need to know that it's applied when just requesting a user or fetching a book list

Reply:
done